components/checkbox/stories/template.js

```diff
@@ -91,3 +91,87 @@ export const Template = ({
 		</label>
 	`;
 };
+
+/* Shows multiple checkboxes in various states of selected, unselected, indeterminate, etc. */
+export const DocsCheckboxGroup = (args, context) => Container({
+	direction: "column",
+	withBorder: false,
+	content: html`
+		${Template({
+			...args,
+			context,
+			iconName: undefined,
+		})}
+		${Template({
+			...args,
+			context,
+			isChecked: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isIndeterminate: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isDisabled: true,
+		})}
+		${Template({
+			...args,
+			context,
+			label: "Checkbox with a long label that should wrap.",
+			customStyles: { "max-inline-size": "200px" },
+		})}
+	`
+});
+
+/* This template group showcases multiple CheckboxGroups in various states of disabled, read-only, invalid, etc. */
+export const AllVariantsCheckboxGroup = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			withBorder: false,
+			direction: "column",
+			heading: "Default",
+			content: html`
+				${DocsCheckboxGroup({
+					...args,
+				}, context)}
+			`
```

Comment:
```suggestion
			content: DocsCheckboxGroup(args, context)
```

This can be simplified